src/main/java/Duke/InputCheckingPackage/InputChecking.java

```diff
@@ -0,0 +1,333 @@
+package Duke.InputCheckingPackage;
+import Duke.Constants.Constants;
+
+import java.util.Objects;
+
+/**
+ * Includes  methods that check if the user input format is valid and will not throw exceptions
+ */
+
+public class InputChecking {
```

Comment:
this class is way too long, consider refactoring the methods and moving some methods somewhere else